### ISSUE TITLE
Flipped Y axis for density map 

### ISSUE THREAD

**Issue by monodera** [pmelchior/skymapper#31]
When I follow the `example.py`, the dec of the density map seems to be flipped. Here is my code and the output:

```python
# load projection and helper functions
import matplotlib.pyplot as plt
import numpy as np
import skymapper as skm


def getCatalog(size=10000, survey=None):
    # dummy catalog: uniform on sphere
    # Marsaglia (1972)
    xyz = np.random.normal(size=(size, 3))
    r = np.sqrt((xyz**2).sum(axis=1))
    dec = np.arccos(xyz[:, 2] / r) / skm.DEG2RAD - 90
    ra = 180 - np.arctan2(xyz[:, 0], xyz[:, 1]) / skm.DEG2RAD

    # survey selection
    if survey is not None:
        inside = survey.contains(ra, dec)
        ra = ra[inside]
        dec = dec[inside]
    return ra, dec


class TestSurvey(skm.survey.Survey):
    def contains(self, ra, dec):
        # simplistic DES like survey
        return (dec < 5) & (dec > -60) & ((ra < 90) | (ra > 300))


if __name__ == "__main__":

    # load RA/Dec from a test catalog
    size = 100000
    survey = TestSurvey()
    ra, dec = getCatalog(size, survey=survey)

    # use Hammer projection
    proj = skm.Hammer(0)

    # construct map:
    # if fig axis is provided, will use it; otherwise will create figure
    # the outline of the map can be styled with kwargs for matplotlib Polygon
    fig = plt.figure(tight_layout=True, figsize=(10, 6))
    ax = fig.add_subplot(111, aspect="equal")

    map = skm.Map(proj, ax=ax)

    # add graticules, separated by 15 deg
    # the lines can be styled with kwargs for matplotlib Line2D
    # additional arguments for formatting the graticule labels
    sep = 15
    map.grid(sep=sep)

    #### 1. plot density in healpix cells ####
    nside = 32
    mappable = map.density(ra, dec, nside=nside)
    cb = map.colorbar(mappable, cb_label="$n$ [arcmin$^{-2}$]")

    # add random scatter plot
    nsamples = 100
    size = 100 * np.random.rand(nsamples)
    map.scatter(ra[:nsamples], dec[:nsamples], s=size, edgecolor="w", facecolor="None")

    # focus on relevant region
    # map.focus(ra, dec)

    # entitle: access mpl figure
    map.title("Hammer projection test")

    # save
    plt.savefig("skymapper-example1_hammer.png")
```

![skymapper-example1_hammer](https://github.com/user-attachments/assets/9c88f74b-e03f-4b11-9ca9-f329340de4e8)

The same happens for the Albers projection as well.

I'm using:
- Python 3.11.9
- Skymapper 0.4.5 (installed via pip)

**Comment by pmelchior** [comment]
Thanks for reporting. This happens because I didn't enforce that `matplotlib.imshow` uses `origin="lower". That is the default on my system, but apparently not on yours. Fixed with the latest commit and in v0.4.6.

**Comment by monodera** [comment]
Thanks! It worked for me. Indeed, the current default value for `image.origin` is actually `upper`.

https://github.com/matplotlib/matplotlib/blob/f8592cbdd79d9b94abea1cacd9cca4f327b19ea0/lib/matplotlib/mpl-data/matplotlibrc#L609
